build/release/dist.js

```diff
@@ -1,190 +1,124 @@
-"use strict";
-
-module.exports = function( Release, files, complete ) {
-
-	const fs = require( "node:fs/promises" );
-	const shell = require( "shelljs" );
-	const inquirer = require( "inquirer" );
-	const pkg = require( `${ Release.dir.repo }/package.json` );
-	const distRemote = Release.remote
-
-		// For local and github dists
-		.replace( /jquery(\.git|$)/, "jquery-dist$1" );
-
-	// These files are included with the distribution
-	const extras = [
-		"src",
-		"LICENSE.txt",
-		"AUTHORS.txt",
-		"dist/package.json",
```

Comment:
How are we generating `package.json` now?

Reply:
This is referring to the `package.json` in the `dist` folder, which sets the module type. Everything in the `dist` folder is included, so it's no longer being specified explicitly.